queries.sql

```diff
@@ -95,4 +97,50 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+-- What animals belong to Melody Pond?
+SELECT animals.name
+FROM animals INNER JOIN owners
+ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name
+FROM animals INNER JOIN species
+ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.full_name, animals.name
+FROM owners LEFT JOIN animals
+ON owners.id = animals.owner_id;
+
+-- How many animals are there per species?
+SELECT species.name, COUNT(animals.name)
+FROM species LEFT JOIN animals
+ON species.id = animals.species_id
+GROUP BY species.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT animals.name, species.name, owners.full_name
+FROM animals INNER JOIN owners
+ON animals.owner_id = owners.id INNER JOIN species
+ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell'
+AND species.name = 'Digimon';
+
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT animals.name, animals.escape_attempts, owners.full_name
+FROM animals INNER JOIN owners
+ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Dean Winchester'
+AND animals.escape_attempts = 0;
+
+-- Who owns the most animals?
+SELECT owners.full_name, COUNT(animals.name)
+FROM owners LEFT JOIN animals
+ON owners.id = animals.owner_id
+GROUP BY owners.full_name
+ORDER BY count DESC;
```

Comment:
- [ ] You have solid queries here :+1: but thinking about the question, don't you think "Most" ownership should return only one owner not he 6 that you have popping at the moment?
![image](https://github.com/mwismann/vet_clinic_2/assets/99715640/f28ab121-7b95-4df7-b620-dc4bb030264e)

I do think you need to check this again. Here's a clue:

```
SELECT owned.full_name
FROM owners AS owned
JOIN animals AS anime on owned.id = anime.owner_id
GROUP BY owned.full_name
ORDER BY COUNT(*) DESC
LIMIT 1;
```
- In fact, what the present query is missing might be the `LIMIT` set to 1. Don't forget to included that query, please :pleading_face:

Reply:
Hello @AbayomiOlaoye, thank you for the feedback!